project-base/.travis.yml

```diff
@@ -19,7 +19,7 @@ jobs:
             script:
                 - sudo apt install ruby
                 - gem install docker-sync
-                - sed -i -r "s#sed -i -E#sed -i -r#" ./scripts/install.sh
+                - sed -i -r "s#sed -i '' -E#sed -i -r#" ./scripts/install.sh
```

Comment:
Is it needed when it's running on Linux?

Reply:
It is, we are simulating istaallation for macOS, where is used sed like `sed -i '' -E` and that needs to be replaced to linux version `sed -i -r`.

Reply:
Oh, I see. I didn't realize that. Nice!